week3/assignment/점메추 🍚/src/styles/globalStyle.js

```diff
@@ -0,0 +1,352 @@
+import { styled, createGlobalStyle } from "styled-components";
+import BMHANNAWOFF from "../assets/fonts/BMHANNAProOTF.woff";
+import BMEULJIRO from "../assets/fonts/BMEULJIRO.woff";
+
+const GlobalStyle = createGlobalStyle`
+* {
+    @font-face {
+        font-family: 'HANNA'; 
+        src: url(${BMHANNAWOFF}) format('woff');
+        font-weight: normal;
+        font-style: normal;
+    }
+    @font-face {
+        font-family: 'EULJIRO'; 
+        src: url(${BMEULJIRO}) format('woff');
+        font-weight: normal;
+        font-style: normal;
+    }
+    box-sizing : border;
+}
+
+#root {
+  max-width: 1080px;
+  
+  margin:0 auto;
+}
+
+html{
+    background: ${({ theme }) => theme.colors.primary};
+}
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+
+  border: 0;
+
+  font-size: 100%;
+  color: ${({ theme }) => theme.colors.white};
+  font-family: 'HANNA';
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+body {
+    line-height: 1;
+}
+ol, ul {
+    list-style: none;
+}
+blockquote, q {
+    quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+button{
+    border:none;
+
+    font-family:"EULJIRO";
+    font-size:${({ theme }) => theme.fontSize.sm};
+
+    cursor:pointer;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+`;
+
+export default GlobalStyle;
+
+export const Section = styled.section`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 auto;
+  max-width: 800px;
+  height: calc(100vh - 70px);
+  text-align: center;
+`;
+
+export const Quetion = styled.h2`
+  font-size: ${({ theme }) => theme.fontSize.lg};
+  font-weight: 100;
+`;
+export const Character = styled.img`
+  width: 150px;
+`;
+export const Answer = styled.div`
+  display: flex;
+  justify-content: center;
+
+  padding: 20px 0 0;
+`;
+export const Balloon = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
이런 요소가 겹치면 겹치는 것만 따로 style 뺄 수 있어요!
 ```suggestion
 ${centerFlex}
```

Reply:
꿀팁 너무 고맙습니당 🖤